src/Microsoft.DotNet.XHarness.Tests.Runners/xUnit/ITestCaseExtenstions.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.XHarness.Tests.Runners.Xunit
+{
+    /// <summary>
+    /// Useful extensions that make working with the ITestCase interface nicer within the runner.
+    /// </summary>
+    public static class ITestCaseExtenstions
```

Comment:
typo in class+filename

Reply:
he, I'm the master of typos.. even when I pay attention, thx